README.md

```diff
@@ -11,6 +11,129 @@ For **questions** or **general comments** regarding the use of this repository,
+
+---
+
+#### Don't use response envelopes by default
+> When the endpoint returns, for example, an array, it should be returned as a plain array, not a dictionary with the array as a value.
+
+
+##### Examples
+###### Good
+
+    [
+      {
+      "like_count": 0,
+      "id": "string",
+      "media_type": "comment",
+      "owner": "string"
+      },
+      {
+        "like_count": 0,
+        "id": "string",
+        "media_type": "comment",
+        "owner": "string"
+      },
+      {
+        "like_count": 0,
+        "id": "string",
+        "media_type": "comment",
+        "owner": "string"
+      }
+    ]
+
+###### Bad
+
+    { "items" : [
+        {
+          "like_count": 0,
+          "id": "string",
+          "media_type": "comment",
+          "owner": "string"
+        },
+        {
+          "like_count": 0,
+          "id": "string",
+          "media_type": "comment",
+          "owner": "string"
+        },
+        {
+          "like_count": 0,
+          "id": "string",
+          "media_type": "comment",
+          "owner": "string"
+        }
+      ]
+    }
+
+##### Reasoning
+If you're calling a **GET** on `/entities` , you're asking for entities, not an object with the entities inside one of its properties. Think about it as a christmas gift. If you ask santa for all the power rangers action figures ( and i'm talking about the 90's ones ) and you only see one box under the tree, you can't know if you're getting just one or all of them wrapped in one gift. _Also,  ALL the power rangers? Couldn't you just ask for ONE toy?_
+
+---
+
+#### Use all HTTP verbs ( Not everything is a GET )
+> Use all the HTTP verbs. If you have to Create something use POST. If you want to Eead Something. if you want to Update something use PATCH or PUT. If you want to Delete something use DELETE. Always remember C.R.U.D. ( Create, Read, Update, Delete) when setting the verb for your resource
+
+
+##### Examples
+###### Good
+
+Creating a comment on a media item: 
+- **POST** `/media/:media_id/comments`
+  Reading comments on a media item:
+- **GET** `/media/:media_id/comments`
+  Updating ( editing ) the comment content on a media item:
+- **PUT**  `/media/:media_id/comments/:id`
+  Deleting a comment on a media item:
+- **DELETE** `/media/:media_id/comments/:id`
+
+###### Bad
+Creating a comment on a media item: 
+- **GET** `/media/:media_id/create_comment?text=comment%20text`
+  Updating ( editing ) the comment content on a media item:
+- **GET**  `/media/:media_id/update_comment?text=comment%20new%20text`
+  Deleting a comment on a media item:
+- **GET** `/media/:media_id/delete_comment?id=comment1`
+
+
+##### Reasoning
+Using the right HTTP Verb on each resource will make clearer what the resource is made for and what we should expect it to do, receive and return. Using always GET is like using your spoon to eat your soup and cut your steak. Just don't.
```

Comment:
Add something about how you can _read_ or _write_ your intentions, e.g.:

* **PUT** /entities/1 : it puts the entity #1 in the server
* **GET** /update_entity?id=1&title=t : it gets all the update_entities with id = 1 and title = t … WAT?